cmd/step-ca/main.go

```diff
@@ -141,7 +142,26 @@ Run the Step CA and read the password from a file - this is useful for
 automating deployment:
 '''
 $ step-ca $STEPPATH/config/ca.json --password-file ./password.txt
-'''`
+'''
+Run the Step CA for the context named _mybiz_ and prompt for password:
+'''
+$ step-ca --context=mybiz
+'''
+Run the Step CA for the context named _mybiz_ and an alternate ca.json file:
+'''
+$ step-ca --context=mybiz other-ca.json
+'''
+Run the Step CA for the context named _mybiz_ and read the password from a file - this is useful for
+automating deployment:
+'''
+$ step-ca --context=mybiz --password-file ./password.txt
+'''
+Run the Step CA for the context that's been selected with step and with a custom password file:
+'''
+$ step context select ssh
+$ step-ca --password-file ./password.txt
+'''
```

Comment:
This should be the first example on the context part, it makes sense to be the default command when contexts are enabled.

You can also remove the passive, just removing `that's been`.